src/image.rs

```diff
@@ -651,6 +651,51 @@ impl Docker {
         self.process_into_value(req).await
     }
 
+    /// ---
+    ///
+    /// # Inspect an Image by contacting the registry
+    ///
+    /// Return image digest and platform information by contacting the dockerd registry
```

Comment:
Can we remove "dockerd", because you've added private registry support

Reply:
should be fixed